docs/classifiers/one-vs-rest.md

```diff
@@ -0,0 +1,25 @@
+<span style="float:right;"><a href="https://github.com/RubixML/ML/blob/master/src/Classifiers/OneVsRest.php">[source]</a></span>
+
+# One Vs Rest
+One Vs Rest is an ensemble learning technique that trains one binary classifier for each potential class label.
```

Comment:
I think we could probably say another sentence or two about One Vs Rest to help give the user a richer idea of what it can do